B2C Implementation Guide (#7310)
auth0/docs: articles/architecture-scenarios/b2c/introduction.md

---
title: Introduction to Consumer Identity and Access Management Implementation
description: High-level overview of the Consumer Identity and Access Management Implementation guide
toc: false
topics:
    - b2c
    - ciam
contentType: concept
useCase:
  - implementation
---

> # The Consumer Identity and Access Management Implementation Guide: An Introduction

Reviewer: ```suggestion
# An introduction to Consumer Identity and Access Management using Auth0
```

> This guide focuses on showing you how to integrate Auth0 within a Consumer Identity and Access Management (CIAM)-style project. Our recommendations are based on real-world customer implementation experience, and we hope that the advice we provide helps you set your project up for success.

Reviewer: When putting the original B2C Welcome together, the consensus was that the language used should be as prescriptive and definitive as possible. So perhaps something like this would work better: 
```suggestion
This guide focuses on showing you how to integrate Auth0 within a Consumer Identity and Access Management (CIAM)-style project. Our recommendations are based on real-world customer implementation experience, and by following the guidance provided you will set your project up for success.
```

> Customers implementing Auth0 for consumer-focused (or Business-to-Consumer (B2C)) projects typically share a common set of goals and objectives. In the sections that follow, we focus on our experiences working with this shared set of expectations. We hope our experience can help you deliver your solution effectively.

Reviewer: ```suggestion
Customers using Auth0 for consumer-focused -  a.k.a. Business-to-Consumer (B2C) - projects typically share a common set of goals and objectives, and in the sections that follow we'll focus on our experiences working with this shared set of expectation to help you deliver your solution effectively and efficiently.
```

Reviewer: I'm also wondering if this needs to be a separate paragraph or if we can't just append to the one above?

Author: We will leave it separate for scanability/readability on smaller devices.

::: note

> The information provided is relevant to **all** project stakeholders. We recommend reading through this in its entirety at least once, even if you've already started your journey with Auth0.

Reviewer: ```suggestion
The information provided is relevant to **all** project stakeholders. We recommend reading through guidance in its entirety at least once, even if you've already started your journey with Auth0.
```

:::

## Ways to integrate Auth0 and how to choose

> There are many different ways Auth0 can be integrated into the CIAM project architecture. Auth0's flexibility comprehensively supports many different use cases. However, keep in mind that not every project requires 100% of the capabilities provided by Auth0.

Reviewer: ```suggestion
There are many different ways Auth0 can be integrated into CIAM project architecture. Auth0's flexibility comprehensively supports many different use cases, however keep in mind that not every project requires 100% of the capabilities provided.
```

![Diagram](#)

> When you embark on your journey to integrate with Auth0, there are many things for you to consider. Knowing what, when, and how best to implement something will help you focus on completing the necessary tasks at the right time. To help you with this, we put together [planning guidance](#) that details our recommended strategies.

Reviewer: ```suggestion
When you embark on your journey to integrate with Auth0, there are many things for you to consider. Knowing what, when, and how best to implement something will help you focus on completing the necessary tasks at the right time. To help you with this, we've put together some [planning guidance](#) that details our recommended strategies.
```

> Throughout the following articles, we have provided numerous recommendations and best practice suggestions. These will be offset appropriately, and any time you across one of these panels, you should stop and consider the advice presented. We are presenting you a set of high-level guidance and best practice recommendations on an ad hoc basis.

Reviewer: ```suggestion
Throughout, we've provided numerous recommendations and best practice suggestions, and any time you come across a panel similar to the one below you should stop and consider the advice presented. These provide you with a set of high-level guidance and best practice recommendations, presented in an ad hoc way.
```

::: note

> You can find detailed guidance regarding specific functionality and use cases in our [documentation](/) or by [speaking with a member of our Professional Services team](/services).

Reviewer: ```suggestion
You can find detailed guidance regarding specific functionality and use cases in our [documentation](/), or by speaking with your Account representative or a member of our [Professional Services team](/services) here at Auth0.
```
